dev-db/postgrest/postgrest-0.4.4.0.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
```

Comment:
see the review for -bin, the same mistakes

Reply:
Thanks for your review @Zlogene, I fixed requested changes.